### DISCUSSION TITLE
custom token selection in web3wallet

### DISCUSSION THREAD

**Discussion by mqaiserkhan** [WalletConnect/walletconnect-monorepo#3712] [Developer Support]
I have a token created on Polygon chain. Now my wallet has both matic token and my custom token. I want to select custom token during selection process using web3wallet. 

Right now only Polygon token is shown during selection process.

I am following this code to setup integration
https://github.com/WalletConnect/web-examples/tree/2.7.8/wallets/react-web3wallet

**Comment by mshahjahan-mufin** [comment]
@mqaiserkhan One of the ways is to select token before you connect via WalletConnectSDK and pass the token contract address over the transaction.

**Reply by mqaiserkhan** [reply]
@shahji-mufin thank you for your reply, But I was unable to find any field in documentation where to pass the contract address, can you please share some example or documentation link which I can follow for this? thank you
